docs/06_how-to-guides/30_key-value-api/kv_table/20_how-to-create-indexes-kv-table.md

```diff
@@ -112,8 +113,9 @@ class [[eosio::contract]] smrtcontract : public contract {
 1. Use the `KV_NAMED_INDEX` with two parameters.
 2. Pass the name of the index as the first parameter. The parameter must be a qualified `eosio::name`. See documentation for the [eosio name restrictions](https://developers.eos.io/welcome/latest/glossary/index#account-name).
 3. Pass the name of the property for which the index is defined as the second parameter.
```

Comment:
Pass the name of the property for which the index is defined as the second parameter.

Not sure I understand this sentence is it:

The second parameter is the name of the property the index is applied to. (or something like)